### ISSUE TITLE
Change shrinkwrap resolutions to use git:// instead of npm registry

### ISSUE THREAD

**Issue by grzegorzjudas** [jeffbski/wait-on#13]
### Problem

Trying to install wait-on on corporate internal network where public npm registry is blocked results in failings due to "resolved" fields referencing public npm registry in shrinkwrap.

Installing with --no-shrinkwrap resolves the problem, but it's not obvious enough for developers and it raises a lot of questions/issues everytime.

### Request
As mentioned in [npm docs](https://docs.npmjs.com/cli/shrinkwrap):
```
The module tree described by the shrinkwrap is reproduced. This means reproducing the structure 
described in the file, using the specific files referenced in "resolved" if available, 
falling back to normal package resolution using "version" if one isn't.
```
and preceding example using git protocol link, I request changing the links to use it instead of https or getting rid of it completely.

**Comment by jeffbski** [comment]
@grzegorzjudas Thanks for mentioning this. I normally don't include a shrinkwrap with my libraries, only my apps, so yes I'll remove that to make it easier for installation.

**Comment by jeffbski** [comment]
Published v2.0.2
